### PR TITLE
Feat impact

### PR DESCRIPTION
**Pull request by JakeatRipple**

### CODE REVIEW

**Comment by jonathanlei** [comment]
could you provide the asana ticket and a preview build here?

**Comment by JakeatRipple** [comment]
Asana 
https://app.asana.com/0/1203315203501656/1203340009247841
Figma 
https://www.figma.com/file/lmaM4BO2xwieSt4md9Eytc/Impact-Page?node-id=1002%3A5779&t=OqSe2JpsPysYrh9i-0
Preview build 
https://jakeatripple.github.io/xrpl-dev-portal/impact.html

File: template/page-impact.html.jinja [hunk 1]
```diff
@@ -8,119 +8,226 @@
...
-                {% trans %}+21.4%{% endtrans %}
-              </p>
-            </div>
-            <p class="numbers">{% trans %}$10.5T{% endtrans %}</p>
-          </div>
-        </div>
+        <h6 class="eyebrow mb-3">A sustainable future</h6>
+        <h2 class="h4 h2-sm mb-8">{% trans %}What makes the XRPL sustainable?{% endtrans %}</h2>
+        <h5 class="longform mb-10">{% trans %}XRPL’s unique consensus mechanism is eco-friendly—it does not require mining to settle transactions. This results in efficiency without sacrificing security, decentralization, or scalability.{% endtrans %}</h5>
+        <p>The trivial amount of energy it does consume is then neutralized with carbon credits through EW Zero, an open-source blockchain decarbonization tool.</p>
```
Comment (mDuo13) [inline template/page-impact.html.jinja]:
```suggestion
        <p>{% trans %}The trivial amount of energy it does consume is then neutralized with carbon credits through EW Zero, an open-source blockchain decarbonization tool.{% endtrans %}</p>
```

File: template/page-impact.html.jinja [hunk 2]
```diff
@@ -8,119 +8,226 @@
...
+      </div>
+
+      <div class="order-2 col-lg-6 px-0 pl-lg-3">
+          <div class="row align-items-center m-0 connect-list"> 
+              <!-- connect list -->
+              <div class="col-12 col-lg-6 p-0 pr-3">
+                  <div class="px-lg-3 pb-3">
+                      <img id="connect-01" src="data:," />
+                      <div class="pt-3">
+                          <h6 class="mb-3">Blog</h6>
```
Comment (mDuo13) [inline template/page-impact.html.jinja]: I think this whole list is missing `{% trans %}` tags as well

File: template/page-impact.html.jinja [hunk 3]
```diff
@@ -8,119 +8,226 @@
...
+      
+      <img src="./img/backgrounds/cta-community-purple.svg" class="cta cta-top-left">
+      <img src="./img/backgrounds/cta-calculator-green.svg" class="cta cta-bottom-right">
+      <div class="z-index-1 position-relative">
+        <div class="d-flex flex-column-reverse">
+          <h2 class="h4 h2-sm mb-10-until-sm mb-8-sm">{% trans %}Featured companies & projects running on the XRP Ledger.{% endtrans %}</h2>
+          <h6 class="eyebrow mb-3">{% trans %}Sustainable Projects{% endtrans %}</h6>
+        </div>
+        <p class="mb-10">{% trans %}Learn more about companies and developers who are using the XRP Ledger to solve interesting problems efficiently and sustainably.{% endtrans %}</p>
+        <a class="btn btn-primary btn-arrow" href="/">{% trans %}See More{% endtrans %}</a>
```
Comment (mDuo13) [inline template/page-impact.html.jinja]: I think `/` is not the intended href here? Maybe you meant to link the new Use Cases page? (If so, don't forget to include `{{currentpage.prefix}}` in the URL so it works in the Japanese target.)

**Review by mDuo13 — CHANGES_REQUESTED** [review]
Mostly looks good, but I noticed a couple issues:

- One missing call-to-action link
- Please remove the old v14.css file before merging

File: template/page-impact.html.jinja (hunk 3 above)
Comment (JakeatRipple) [reply template/page-impact.html.jinja]: changed to link to usecases.

File: template/page-impact.html.jinja (hunk 2 above)
Comment (JakeatRipple) [reply template/page-impact.html.jinja]: added trans tag for text

**Review by jonathanlei — APPROVED** [review]
LGTM

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**